state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -267,6 +283,10 @@ pub mod pallet {
 	#[pallet::getter(fn maximum_swap_amount)]
 	pub type MaximumSwapAmount<T: Config> = StorageMap<_, Twox64Concat, Asset, AssetAmount>;
 
+	/// The fixed fee charged for opening a channel, in Flipperinos.
+	#[pallet::storage]
+	pub type ChannelOpeningFee<T: Config> = StorageValue<_, T::Amount, ValueQuery>;
```

Comment:
rather than require another round of governance actions (which we already have to do for the upgrade), we could have a migration to set this to some const

Reply:
We could but I didn't want to unilaterally decide on a value - might require some discussion.